scripts/openclaw_bridge.py

```
@@ -0,0 +1,346 @@
+def parse_agent_file(path: Path) -> AgentDefinition | None:
+    """Parse a single ai-agents markdown file with YAML frontmatter.
+
+    Args:
+        path: Path to the agent definition file.
+
+    Returns:
+        AgentDefinition if valid, None if the file lacks required fields.
+    """
+    try:
+        post = frontmatter.load(str(path))
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)